content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -250,20 +245,20 @@ Here is an example manifest:
 
 To use a gRPC probe, `port` must be configured. If you want to distinguish probes of different types
 and probes for different features you can use the `service` field.
-You can set `service` to the value `liveness` and make your gRPC Health Checking endpoint
-respond to this request differently then when you set `service` set to `readiness`.
+You can set `service` to the value `liveness` and make your gRPC health checking endpoint
```

Comment:
gRPC Health Checking is a proper noun. See https://github.com/grpc/grpc/blob/master/doc/health-checking.md

Reply:
Fine.